src/UI/src/app/file-manager/asset-details/asset-details.component.ts

```diff
@@ -70,8 +78,28 @@ export class AssetDetailsComponent implements OnInit {
     "ppt",
     "html",
     "txt",
+    "bmp",
+    "svg",
+    "ico",
+    "gif",
   ];
 
+  private supportFileTypes2 = {
```

Comment:
remove supportedFileTypes with supportedFileTypes2(bad naming convention)  and adjust code accordingly.